spec/features/tanks/show_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'rails_helper'
+
+describe "When I visit the tank Show page" do
+  let(:user) { create(:user) }
+
+  before do
+    sign_in user
+  end
+
+  it "Then I see information of a specific tank" do
+    tank = FactoryBot.create(:tank)
+
+    visit tank_path(tank)
+
+    expect(page).to have_content(tank.name)
+    expect(page).to have_content(tank.facility.name)
```

Comment:
Thoughts on adding a delegation here to help (somewhat) mitigate [LOD](https://en.wikipedia.org/wiki/Law_of_Demeter) violations here and in the future?

Reply:
I like that idea, something like a `facility_name`?

Reply:
Yup, I was thinking something along the lines of...

```ruby
delegate :name, to: :facility, prefix: true
```

on the `Tank`